codecs/rust.Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN wget -qO- https://github.com/rustwasm/wasm-pack/releases/download/v0.9.1/was
 FROM $RUST_IMG AS rust
 ARG RUST_IMG
 RUN rustup target add wasm32-unknown-unknown
-RUN if [[ $RUST_IMG = rustlang/rust:* ]] ; then rustup component add rust-src ; fi
+RUN case $RUST_IMG in rustlang/rust@*) rustup component add rust-src; esac
```

Comment:
wtf?

Reply:
If you really want to know, turns out that syntax is bash-ism and not supported by regular `sh` which Docker uses. This is the recommended alternative.